ironfish/src/blockchain/database/transactions.ts

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import type { IDatabaseEncoding } from '../../storage/database/types'
+import bufio from 'bufio'
+import { Transaction } from '../../primitives/transaction'
+import { WorkerPool } from '../../workerPool'
+
+export type TransactionsValue = {
+  transactions: Transaction[]
+}
+
+export class TransactionsValueEncoding implements IDatabaseEncoding<TransactionsValue> {
+  constructor(readonly workerPool: WorkerPool) {}
```

Comment:
This doesn't need the worker pool now

Reply:
https://github.com/iron-fish/ironfish/pull/1265